docs/modules/concepts/pages/processing/elements/index.adoc

```diff
@@ -0,0 +1,13 @@
+= Processing Elements
+:navtitle: Elements
+
+== Overview
+
+If we take the view of a mail server as a mail processing engine, then
+we can list some of the elemental processes that a typical mail server
+will likely perform:
+
+ * xref:processing/elements/receiving.adoc[Receiving] email from a remote system
+ * xref:processing/elements/queuing.adoc[Queuing] and spooling emails
+ * xref:processing/elements/chaining.adoc[Chaining] processes
```

Comment:
I don't understand what "chaining means" maybe we should keep "Mail Processing' wording to fit in the James wording?"

Reply:
I am having trouble with this one. In order to explain what "processing" means, I have broken it down into the "elements" that I was able to identify. If I try to change "chaining" to "email processing", not only does it take the level of abstraction up to the same level I am trying to define (which, in order to define requires breaking it down), it also creates a circular definition.

I will try to actually fill in the "Chaining" section. Perhaps after reading it you could comment again? If you still think the same way, I will try to to conceive of a different approach.

Reply:
@chibenwa, I just added the content to the "chaining" section that this link points to. Please let me know if you still think this is problematic.